mobilecoind/src/config.rs

```diff
@@ -146,7 +144,8 @@ impl PeersConfig {
                 .name_prefix("RPC".to_string())
                 .build(),
         );
-        let peers = self.create_peers(measurement, grpc_env, logger.clone());
+        let measurements: Vec<Measurement> = vec![measurement.into()];
```

Comment:
Same as before, this can simply be `&[measurement.into()]` or `let measurements = [measurement.into()];` and later `&measurements[..]`.